### PR TITLE
Replace error redirects

### PR DESCRIPTION
**Pull request by yousiefc**
# Description

This PR includes the following proposed change(s):

- Replacing more of the error redirects with simple toasts

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

```bash
yarn lint
yarn test -u
```

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js [hunk 1]
```diff
@@ -11,10 +11,10 @@ import ConfirmationPopup, {
   Sidecard,
 } from "shared-components";
 import validator from "validator";
+import { Toast } from "../../toast/Toast";
```
Comment (TayGov) [inline src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js:14]: 🍞 🥂

File: src/frontend/efiling-frontend/src/domain/package/package-confirmation/PackageConfirmation.js [hunk 2]
```diff
@@ -51,6 +52,7 @@ export default function PackageConfirmation({
   const [submissionFee, setSubmissionFee] = useState(null);
   const [showPayment, setShowPayment] = useState(false);
   const [showUpload, setShowUpload] = useState(false);
+  const [show, setShow] = useState(false);
```
Comment (KDNeufeld) [inline src/frontend/efiling-frontend/src/domain/package/package-confirmation/PackageConfirmation.js:55]: Maybe rename "show" variable to "showToast".   Helps clarify the code if we're showing more than one thing.

File: src/frontend/efiling-frontend/src/domain/package/package-confirmation/__tests__/FileListItem.test.js [hunk 3]
```diff
@@ -28,14 +30,14 @@ describe("FileListItem Testsuite", () => {
     await waitFor(() => {});
 
     expect(service.downloadFileByName).toHaveBeenCalled();
-    expect(error.errorRedirect).not.toHaveBeenCalled();
+    expect(queryByText(toastMessage)).not.toBeInTheDocument();
```
Comment (KDNeufeld) [inline src/frontend/efiling-frontend/src/domain/package/package-confirmation/__tests__/FileListItem.test.js:33]: Nice.  good work.

File: src/frontend/efiling-frontend/src/domain/package/package-confirmation/PackageConfirmation.js (hunk 2 above)
Comment (yousiefc) [reply src/frontend/efiling-frontend/src/domain/package/package-confirmation/PackageConfirmation.js]: good call just made the change

**Review by KDNeufeld — APPROVED** [review]
Great work!

**Pull request merged**